raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -1220,7 +1265,11 @@ def handle_onchain_secretreveal(
     return iteration
 
 
-def handle_unlock(mediator_state, state_change: ReceiveUnlock, channelidentifiers_to_channels):
+def handle_unlock(
+        mediator_state: MediatorTransferState,
+        state_change: ReceiveUnlock,
+        channelidentifiers_to_channels: typing.ChannelMap,
+)-> TransitionResult:
```

Comment:
s/)-> /) -> /

Reply:
Ah nice! I would have expected flake to have caught this